### PR TITLE
BaSP-MR-158: Member profile

### PR DESCRIPTION
**Pull request by Gonzarey96**
Member profile screen that displays the user's data and allows to modify it based on business logic.

- Edit aggregate mode
- Memberships cannot be modified
- Hide and show passwords button
- Figma styles

### CODE REVIEW

**Comment by Federico-Peralta** [comment]
[[Member : Profile] Create new member profile screen](https://trello.com/c/JMtiquAz/158-member-profile-create-new-member-profile-screen)

File: src/Components/Member/Profile/index.jsx [hunk 1]
```diff
@@ -1,7 +1,237 @@
...
+                <button
+                  className={styles.toggleButton}
+                  type="button"
+                  onClick={() => setShowPassword(!showPassword)}
+                >
+                  {showPassword ? <FaEyeSlash /> : <FaRegEye />}
+                </button>
+              </div>
+            </div>
+
```
Comment (AleMac17) [inline src/Components/Member/Profile/index.jsx]: We shouldn't leave blank space

**Review by fernandofernandezuy — APPROVED** [review]
Looks good, great job Gonza!

**Review by marcohzo — APPROVED** [review]

**Comment by nazarenapantanetti** [comment]
Tested and approved! A suggestion was made in the following ticket: https://trello.com/c/2h20i2h0/189-suggestion-button-color-consistency

**Pull request merged**